src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
+    }
+
+    @Override
+    public void stopWork() {
+        return "certain machines stopped working";
```

Comment:
Similarly to 'doWork', the 'stopWork' method should print out a message instead of returning a string. Use System.out.println to indicate that the Bulldozer has stopped working.